docs/detections/alerts-view-details.asciidoc

```diff
@@ -65,8 +65,26 @@ If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription
 * *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
 
 [discrete]
-[[enriched-data-overview]]
-==== Enriched data on alerts
+[[alerts-enrich-host-user-risk-score]]
+==== Alerts enriched with user and host risk scores
+Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels can be `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.
+
+NOTE: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to <<host-risk-score>> and <<user-risk-score>> to learn more.
```

Comment:
Note to self: Might need to update these anchors one more time after host and user risk score install docs are updated. ATM, an anchor exists for the user risk score install docs, but one isn't available for the host risk score install docs.

Reply:
Would updating these anchors break any existing URLs? I don't know the architecture exactly, but I'm a little paranoid about renaming anchors after I burned myself a few releases back when I broke some URLs and had to request some redirects for the old -> new URLs.

Reply:
I'm not actually changing the anchor names. I left this note for myself in case an anchor was added to [this section](https://www.elastic.co/guide/en/security/master/host-risk-score.html#_deploy_host_risk_score) of the host risk score docs. Since the section doesn't have anchor atm, I'm pointing to the [main topic](https://www.elastic.co/guide/en/security/master/host-risk-score.html). 

I might actually resolve this comment early though. The install instructions for both risk score features are fairly easy to find in the linked topics, so adding new anchor links to my topic aren't all that necessary.

Reply:
Cool, thanks for explaining!